src/status_im/ui/screens/navigation.cljs

```diff
@@ -135,7 +135,7 @@
 (handlers/register-handler-fx
  :navigate-to-clean
  (fn [cofx [_ view-id params]]
-   (navigate-to-clean cofx view-id params)))
+   (navigate-to-cofx cofx view-id params)))
```

Comment:
no clean anymore?

Reply:
yes, it crashes now and seems to work fine with `navigate-to-cofx`. Will see how tests will pass